Scripts/Python/xDialogStartUp.py

```diff
@@ -340,25 +344,35 @@ def OnGUINotify(self,id,control,event):
                     try:
                         playerName == playerNameW
                     except:
-                        errorString = "Error, invalid Name. Please enter another."
+                        errorString = "Error: invalid Name. Please enter another."
```

Comment:
This needs to be moved to a loc file. I realize that the rest of the script is hardcoding the messages, but we should avoid doing naughty things like this.

Reply:
Edit: this comment was made on the wrong line. You only need to put your new messages in a loc file. Sorry about that!